packages/transform/rollup.config.ts

```diff
@@ -0,0 +1,33 @@
+import { resolve } from 'path';
+import cleaner from 'rollup-plugin-cleaner';
+import typescript from 'rollup-plugin-typescript2';
+
+export default {
+  input: 'src/index.ts',
+  output: [
+    {
+      file: './dist/index.js',
+      format: 'cjs',
+      exports: 'named',
+      sourcemap: true
+    },
+    {
+      file: './dist/index.mjs',
+      format: 'es',
+      exports: 'named',
+      sourcemap: true
+    },
+    {
+      file: './dist/index.umd.js',
+      format: 'umd',
+      name: 'Transform',
+      sourcemap: true,
+      globals: {
+        '@joshdb/provider': 'JoshProvider',
+        '@sapphire/utilities': 'SapphireUtilities'
+      }
+    }
```

Comment:
```suggestion
    }
```
We no longer support umd bundles

Reply:
Duplicated from `AutoEnsureMiddleware` in the beginning, updated again using it again.